test/e2e/const.go

```diff
@@ -42,6 +42,12 @@ var (
 	//go:embed data/capi-operator/azure-provider.yaml
 	AzureProvider []byte
 
+	//go:embed data/capi-operator/kubeadm-bootstrap-provider.yaml
+	KubeadmBootstrapProvider []byte
```

Comment:
can these providers be installed in the same way as we do for Docker https://github.com/rancher/turtles/blob/main/test/e2e/suites/import-gitops-v3/import_gitops_v3_test.go#L43?

Reply:
Sure, I wasn't sure whether we wanted to keep things isolated or whether it's ok installing CAPD as well. I will align the tests.